front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -22,9 +25,35 @@ const ScenarioDescription = ({
 }: ScenarioDescriptionProps) => {
   const { t } = useTranslation('operationalStudies/scenario');
   const { openModal } = useModal();
+  const [isOpenedDescription, setIsOpenedDescription] = useState<boolean>(false);
+  const [width, setWidth] = useState<number>(0);
+  const ref = useRef<HTMLInputElement | null>(null);
+  const descriptionRef = useRef<HTMLDivElement | null>(null);
+
+  const showDescription = () => {
+    setIsOpenedDescription(!isOpenedDescription);
+  };
```

Comment:
Seems like this function may hide the description instead of showing it. Maybe `toggleDescription` would be a better name?

Reply:
Done